File: Algorithms/Python3/00058-Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s = s.rstrip()
        size = len(s)
        count = 0
        for i in reversed(range(size)):
            if s[i] == ' ':
                return count
            count += 1

        return count

        # s = s.strip()
        # words = s.split(' ')
        # return len(words[-1])

File: Algorithms/C++/00058-Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int r = s.size() - 1;

        while(r >= 0){
            if(s[r] == ' ') r--;
            else break;
        }

        int count{0};
        while(r >= 0){
            if(s[r] != ' '){
                count++;
                r--;
            } else{
                break;
            }
        }

        return count;
    }
};